content/en/docs/setup/upgrade/in-place/index.md

```diff
@@ -35,7 +35,7 @@ Before you begin the upgrade process, check the following prerequisites:
 
 {{< warning >}}
 Traffic disruption may occur during the upgrade process. To minimize the disruption, ensure
-that at least two replicas of each component (except Citadel) are running. Also, ensure that
+that at least two replicas of each component are running. Also, ensure that
```

Comment:
What are the components?  Do we just mean istiod, or do we also mean gateways?

Reply:
I think this is in reference to the deployments. Only istiod is shipped.

Reply:
could we then say "two replicas of Istiod"?